gulpfile.js

```diff
@@ -121,7 +121,6 @@ function writeServiceWorkerFile() {
       `${rootDir}/js/*.js`,
       `${rootDir}/css/*.css`,
       `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
-      `${rootDir}/bower_components/highlight/highlight.js`,
```

Comment:
Ugh, we still have to figure out how to fix this. I'm ok with removing it for now, but maybe uncommenting it instead of deleting?
@arthurevans who has been looking into why it's hosed